### PR TITLE
Ddids 1124 modify angular template

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Updated angular template to match react template

### CODE REVIEW

File: src/app/basic-form/basic-form-success.component.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
```
Comment (ArakTaiRoth) [inline src/app/basic-form/basic-form-success.component.ts]: You can completely remove lines 2 and 3 here

File: src/app/basic-form/basic-form-success.component.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-basic-form-success',
+  templateUrl: './basic-form-success.component.html',
+  styleUrls: ['./basic-form.component.css'],
+})
+
+export class BasicFormSuccessComponent {
+    constructor(private fb: FormBuilder, private location: Location, private router: Router) { }
```
Comment (ArakTaiRoth) [inline src/app/basic-form/basic-form-success.component.ts]: Remove FormBuilder and Location use here, these aren't being used

**Comment by ArakTaiRoth** [comment]
This doesn't show up when submitting the form, like it does in the React template:
image

File: src/app/basic-form/basic-form-success.component.ts (hunk 1 above)
Comment (Radha-Alberta) [reply src/app/basic-form/basic-form-success.component.ts]: Done

File: src/app/basic-form/basic-form-success.component.ts (hunk 2 above)
Comment (Radha-Alberta) [reply src/app/basic-form/basic-form-success.component.ts]: Done

**Pull request merged**